transformer/denylist.py

```diff
@@ -10,7 +11,11 @@ def get_empty() -> Denylist:
 
 
 def from_file() -> Denylist:
-    denylist_file = f"{os.getcwd()}/.urlignore"
+    if os.path.exists(f"{os.getcwd()}/.urlignore"):
+        logging.warning(
+            "Legacy .urlignore file detected - it will not be used! Rename it to '.ignore' if you want to use it."
```

Comment:
I’d also emphasize the change in behavior between `.urlignore` and `.ignore`:
```suggestion
            "Legacy .urlignore file detected - it will not be used! Rename it to '.ignore' if you want to use it, and read about the difference in the documentation."
```